### ISSUE TITLE
duplicated check for Imports in DESCRIPTION but not in NAMESPACE

### ISSUE THREAD

**Issue by LiNk-NY** [Bioconductor/BiocCheck#212]
`BiocCheck` seems to provide a duplicate check of any packages listed in the `Imports:` field but not in the `NAMESPACE`. This is also reported by `R CMD check` (see below). I am in favor of removing the check in `BiocCheck`. 

Note that packages can use `::` exclusively for all symbols in any namespace in the `Imports:` field and this will trigger the `WARNING` / `NOTE`.

In `BiocCheck`:

```r
* Checking DESCRIPTION/NAMESPACE consistency...
    * WARNING: Import utils in NAMESPACE as well as DESCRIPTION.
```

In `R CMD check`: 

```r
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: ‘utils’
  All declared Imports should be used.
```

Does this indeed look redundant? @mtmorgan 

Starting line: https://github.com/Bioconductor/BiocCheck/blob/fa4f261c55ce02a2519c2c44aed91eba0c0ff6d3/R/checks.R#L547

**Comment by mtmorgan** [comment]
This looks redundant to me, too
